src/psycop_model_training/model_eval/base_artifacts/tables/table_1.py

```diff
@@ -0,0 +1,305 @@
+"""Code for generating a table 1."""
+import warnings
+from pathlib import Path
+from typing import Optional, Union
+
+import pandas as pd
+import wandb
+
+from psycop_model_training.model_eval.base_artifacts.tables.tables import output_table
+from psycop_model_training.model_eval.dataclasses import EvalDataset
+from psycop_model_training.utils.utils import bin_continuous_data
+
+
+class Table1:
```

Comment:
```suggestion
class DescriptiveStatsTable:
```

Mostly a preference - table_1 mostly makes sense if you're in the medical field, I think, and could easily be an appendix if in another field. Would rename the file as well.